sw/ground_segment/cockpit/shapes.ml

```diff
@@ -0,0 +1,141 @@
+(*
+ * This file is part of paparazzi.
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ *
+ *)
+
+open Latlong
+
+type shdata = {
+  shid : string;
+  shlinecolor : string;
+  shfillcolor : string;
+  shtype : float;
```

Comment:
type and status should not be float, at least int or even better a specific type